images/qemu-crinit-x86_64/root/etc/apt/sources.list

```diff
@@ -0,0 +1,5 @@
+deb http://linux.elektrobit.com/eb-corbos-linux/1.1.1 ebcl prod
+deb http://linux.elektrobit.com/eb-corbos-linux/1.1.1 ebcl dev
+deb http://linux.elektrobit.com/eb-corbos-linux/1.1.1 ebcl all
+deb http://linux.elektrobit.com/eb-corbos-linux/1.1.1 ebcl main
+deb http://de.archive.ubuntu.com/ubuntu jammy main restricted universe multiverse
```

Comment:
Is this change required?
And if so why only for qemu-crini-x86_64 but not for systemd or aarch64?

Reply:
It's an improvement, and it should be added for all example images, just had no time by now to do it.

Reply:
So it is required also in https://gitext.elektrobitautomotive.com/EB-Linux/nautilos/pull/241, isn't it?